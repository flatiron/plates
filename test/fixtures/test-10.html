<br><img class="logo"><input name="first_name"/><input name="last_name" />