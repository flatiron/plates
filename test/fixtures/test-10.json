{
  "url": "/some/image.png",
  "fname": "Raul",
  "lname": "Julia"
}