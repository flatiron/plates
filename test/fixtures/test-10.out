<br><img class="logo" src="/some/image.png"><input name="first_name" value="Raul"/><input name="last_name" value="Julia" />